app/app.go

```diff
@@ -336,7 +352,8 @@ func NewInitApp(
 	// During begin block slashing happens after distr.BeginBlocker so that
 	// there is nothing left over in the validator fee pool, so as to keep the
 	// CanWithdrawInvariant invariant.
-	app.mm.SetOrderBeginBlockers(distr.ModuleName,
+	app.mm.SetOrderBeginBlockers(mint.ModuleName,
```

Comment:
Do the minted tokens get distributed in the same block ?

Reply:
Yes, they get sent to the fee collector every block